src/main/java/com/alvarium/annotators/AnnotatorFactory.java

```diff
@@ -50,6 +51,9 @@ public Annotator getAnnotator(AnnotatorConfig cfg, SdkInfo config, Logger logger
         return new VulnerabilityAnnotator(vulnCfg, hash, signature, logger);
       case SOURCE:
         return new SourceAnnotator(hash, signature, logger);
+      case SBOM:
```

Comment:
A general comment -- Once this PR is merged, be sure to add this key to the constants in the Go SDK. See the following PR.

https://github.com/project-alvarium/alvarium-sdk-go/pull/50/files

Reply:
Will prepare a PR to merge right after this one.